mantidproject/mantid
# IndirectDataReduction - post plot and save

The plotting and saving of the results workspace has been changed to after the execution of the algorithm for all Indirect Data Reduction tabs, apart from EnergyTransfer.

**To Test:**
Files to test with: \olympic\babylon5\Public\Louise McCann\DataReduction
For each tab load relevant files and run algorithm. Check each plot option plots a suitable graph and that the appropriate workspaces have been saved in your save directory.
- [x] ISIS Energy Transfer
- [x] ISIS Calibration
- [x] ISIS Diagnostic
- [x] Transmission
- [x] Symmetrise
- [x] S(Q, w)
- [x] Moments

Fixes #17023 



---
#### Reviewer

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):
##### Code Review
##### Functional Tests
- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
